README.md

```diff
@@ -53,6 +53,9 @@ make docker_build_and_up
 
 Then, you can access the simulator with http://localhost:3000
 
+Note: 4GB of memory should be allocated to Docker.
```

Comment:
We haven't exactly figured out the cause. This problem may occur only in docker for mac. (I mean it may not always require 4GB momory)
So, lMO it is better to change the wording to something a little more ambiguous.

like: 
```
Note: Insufficient memory allocation sometimes causes problems in building the image.
Please allocate enough memory in that case.
```

Reply:
4GB sounds like a lot. What is consuming that? the frontend or the backend?

Reply:
Failing to build an Image.

I had a problem with 2GB memory allocation.
With 4GB, it did not happen.